docs/source/learn/examples/dimensionality.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>use <code>{class}'arviz.InferenceData'</code>  (changing apostrophes for backticks) so that it is rendered as a link to arviz docs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5437/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>